telesto/segmentation/app.py

```diff
@@ -0,0 +1,210 @@
+import base64
+import io
+import os
+import socket
+import threading
+import time
+from dataclasses import asdict
+from importlib import import_module
+from typing import Deque, List
+from uuid import uuid4
+import json
+from collections import deque
+
+import PIL
+from PIL.Image import Image
+import falcon
+
+from telesto.segmentation import SegmentationObject, DataStorage
+from telesto.logger import logger
+from telesto.segmentation.model import DummySegmentationModel, SegmentationModelBase
+
+
+API_CONFIG = {
+    "name": "Nucleus instance segmentation",
+    "desc": "This API provides endpoints for segmenting nuclei on fluorescent images",
+    "input": {
+        "type": "png",
+        "palette": "RGB24",
+        "encoding": "base64",
+        "max_size": "5120",
+    },
+    "output": {
+        "type": "json",
+        "palette": "GREY8",
+        "encoding": "plain",
+    },
+    "classes": [
+        "bg", "fg"
+    ],
+}
+
+API_DOCS = {
+    "name": API_CONFIG["name"],
+    "description": API_CONFIG["desc"],
+    "endpoints": [
+        {
+            "path": "/",
+            "method": "GET",
+            "name": "Status endpoint",
+            "description": "Returns status of the API",
+        },
+        {
+            "path": "/docs",
+            "method": "GET",
+            "name": "Information endpoint",
+            "description": "Returns this information",
+        },
+        {
+            "path": "/jobs/",
+            "method": "POST",
+            "request_body": {
+                "image": "<str>",
+            },
+            "image_format": {
+                **API_CONFIG["input"],
+            },
+            "response_body": {
+                "job_id": "<UUID>"
+            }
+        },
+        {
+            "path": "/jobs/<job_id>",
+            "method": "GET",
+            "response_body": {
+                "objects": [
+                    {
+                        "class_i": "<int>",
+                        "x": "<int>",
+                        "y": "<int>",
+                        "w": "<int>",
+                        "h": "<int>",
+                        "data": "<str>"
+                    }
+                ],
+            },
+            "object_data_format": {
+                **API_CONFIG["output"],
+            },
+            "classes": API_CONFIG["classes"],
+        }
+    ]
+}
```

Comment:
Shouldn't these belong to an example?

Reply:
`API_CONFIG` should be shared with or provided by telesto-api but it has nothing to do with model examples.